resources/definitions/fdmprinter.def.json

```diff
@@ -1273,8 +1273,8 @@
                 },
                 "optimize_wall_printing_order":
```

Comment:
I think leaving this called `optimize_` will create confusion later on. It will certainly be raised once or twice during a discussion.
However chancing this requires multiple additional changes so I'm not sure what will be the best approach.
Can you explain why you left it called `optimize_`?

Reply:
I think we should change it, since keeping the same name for something that will change the "original" behaviour could break backward compatibility if somebody imports an "old" 3mf file

Reply:
Backwards compatibility. It still gives more or less the same behaviour as optimize ... it just isn't optimized any more. (Hence also the switch). I'll put it in Cura & Cookies.

Reply:
Also it would require a version upgrade to actually rename the 'internal name'.

Reply:
True indeed, lets discuss in Cura & Cookies